include/rocksdb.h

```diff
@@ -149,14 +161,38 @@ struct rocksdb_column_family_options_s {
 
   /** @since 2 */
   rocksdb_filter_policy_t filter_policy;
+
+  /** @since 3 */
+  bool optimize_filters_for_hits;
+
+  /** @since 3 */
+  int num_levels;
+
+  /** @since 3 */
+  int max_write_buffer_number;
+
+  /** @since 3 */
+  bool paranoid_file_checks;
```

Comment:
I'm unsure if we'll ever want to enable this as it will have a negative impact on performance.